dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-email/src/test/java/org/apache/dolphinscheduler/plugin/alert/email/MailSenderTest.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.dolphinscheduler.plugin.alert.email;
```

Comment:
Missing licenses header.

Reply:
Thanks for review, the license header has been added now, please review again.